examples/plot_theta_square.py

```diff
@@ -74,42 +76,39 @@
         # Calulate hillas parameters
         # It fails for empty pixels   
         try:
-            hillas_params[tel_id] = hillas_parameters(camgeom, cleaned_image)
+           hillas_params[tel_id] = hillas_parameters(camgeom, cleaned_image)
```

Comment:
indentation in this project seems to be 4 spaces ... here it was changed to 3 spaces, why?